src/js/tracks/text-track-settings.js

```diff
@@ -435,9 +461,11 @@ class TextTrackSettings extends Component {
    * @private
    */
   createElControls_() {
+    const defaultsDescription = this.localize('restore all settings to the default values');
     const defaultsButton = createEl('button', {
       className: 'vjs-default-button',
-      textContent: this.localize('Defaults')
+      title: defaultsDescription,
+      innerHTML: `${this.localize('Defaults')}<span class='vjs-control-text'> - ${defaultsDescription}</span>`
```

Comment:
@OwenEdwards @mmcc brought up that maybe 'Reset' would be a better name for this button, thoughts?